src/interfaces/navigator.ts

```diff
@@ -0,0 +1,28 @@
+import { NavigatorConnectionVariants } from "../enumerations/navigator-connection-variants";
```

Comment:
How do we feel about pluralizing the file since there's multiple interfaces being exported? Additionally, I don't think I saw these being re-exported from the index file - just a heads up

Reply:
👍 Would also be good with that! Most of our interfaces are broken out separately